src/themes/minechat.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
```

Comment:
This font is a pretty good substitute! If we wanted to be a little more authentic, it turns out there are [some pens out there](https://codepen.io/Daltontastic/pen/YWVadG) that feature the Minecraft font as a `data:font` URL. I think it's up to you if you'd like to use that.

Reply:
I like that  font better, I'm updating that now